src/Features/Core/Portable/UnusedReferences/UnusedReferencesService.cs

```diff
@@ -0,0 +1,199 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Composition;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+
+namespace Microsoft.CodeAnalysis.UnusedReferences
+{
+    [ExportWorkspaceService(typeof(IUnusedReferencesService), ServiceLayer.Default), Shared]
```

Comment:
Does this need to be a workspace service or can it just be a bunch of static methods?

Reply:
Nope, no need to be a workspace service. Should I keep the name or change it to something like "UnusedReferenceAnalyzer"?